examples/js/loaders/FBXLoader.js

```diff
@@ -4061,7 +4065,6 @@
 			// The first three data sizes depends on version.
 			var endOffset = ( version >= 7500 ) ? reader.getUint64() : reader.getUint32();
 			var numProperties = ( version >= 7500 ) ? reader.getUint64() : reader.getUint32();
-			var propertyListLen = ( version >= 7500 ) ? reader.getUint64() : reader.getUint32();
```

Comment:
Why did you remove this line?
I don't think we should remove...

Reply:
Ah, probably you did that because `propertyListLen` is unused, correct?
But the line is necessary to move the pointer forward in the binary file.

Reply:
Uh oh! I'll revert.

Reply:
> But the line is necessary to move the pointer forward in the binary file.

Sry guys! I've overlooked that...

Reply:
No worries! 😊